### PR TITLE
[Task]: Setup Phpstan #4

### PR DESCRIPTION
**Pull request by kingjia90**
Resolves partially #3

### CODE REVIEW

**Comment by aashan10** [comment]
@dvesh3 please close this in favor of #5  as it is working fully now

File: phpstan-baseline.neon [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+		-
+			message: "#^Call to method getTotalNetPrice\\(\\) on an unknown class Pimcore\\\\Model\\\\DataObject\\\\OnlineShopOrder\\.$#"
+			count: 2
+			path: src/PaymentManager/Payment/Mpay24Seamless.php
+		-
+			message: "#^Call to method getItems\\(\\) on an unknown class Pimcore\\\\Model\\\\DataObject\\\\OnlineShopOrder\\.$#"
+			count: 1
+			path: src/PaymentManager/Payment/Mpay24Seamless.php
+		-
+			message: "#^Call to method getPriceModifications\\(\\) on an unknown class Pimcore\\\\Model\\\\DataObject\\\\OnlineShopOrder\\.$#"
```
Comment (dvesh3) [inline phpstan-baseline.neon]: we should add classes to the ci instead of supressing the errors.

**Comment by aashan10** [comment]
@dvesh3 this one is done. Please have a look

File: .github/ci/files/var/classes/DataObject/OnlineShopOrder/PaymentProvider.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Pimcore\Model\DataObject\OnlineShopOrder;
+
+use Pimcore\Model\DataObject\Exception\InheritanceParentNotFoundException;
+
+class PaymentProvider extends \Pimcore\Model\DataObject\Objectbrick {
+
+protected $brickGetters = ['PaymentProviderPayPalSmartButton'];
```
Comment (dvesh3) [inline .github/ci/files/var/classes/DataObject/OnlineShopOrder/PaymentProvider.php]:
```suggestion
protected $brickGetters = ['PaymentProviderMpay24'];
```
Please change at other places as well

**Pull request merged**